<launch>
 <arg name = "semantic_files" value = "$(find adaptive_controller)/asp/semantic_map.lp" />
 <arg name="robot_name" default="robot0" />

<node pkg="adaptive_controller" type="contextNode.py" name="context_middleware" args="$(arg semantic_files)" respawn="true">
   <param name="robot_name" value="$(arg robot_name)"/>
 </node>

 <node pkg="adaptive_controller" type="testInput.py" name="test_context_input" output="screen" respawn="true">
   <param name="robot_name" value="$(arg robot_name)"/>
 </node>

 <!--node pkg="adaptive_controller" type="testOutput.py" name="test_context_output" output="screen" respawn="true">
   <param name="robot_name" value="$(arg robot_name)"/>
 </node-->
</launch>
